### PR TITLE
Migrate to net track2 for some resource-manager readmes

### PR DESCRIPTION
**Pull request by fengzhou-msft**
For the dotnet SDKs that have released track 2 versions, change the swagger-to-sdk settings to only keep azure-sdk-for-net-track2.

<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [ ] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [ ] SDK of Python
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following apply to the PR so that label "WaitForARMFeedback" will be added automatically to begin ARM API Review. Failure to comply may result in delays to the manifest.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/connectedvmware/resource-manager/readme.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-js
   - repo: azure-sdk-for-net
   - repo: azure-sdk-for-ruby
-  - repo: azure-sdk-for-net
     after_scripts:
       - bundle install && rake arm:regen_all_profiles['azure_mgmt_connectedvmware']
```
Comment (fengzhou-msft) [inline specification/connectedvmware/resource-manager/readme.md:60]: This one is not supported by track 2 yet, just remove the duplicate setting which steals the after script that should belong to ruby.

**Comment by fengzhou-msft** [comment]
The net track2 failure for insights and machinelearningservices are expected as these 2 are still the old track 2 packages. Azure.ResourceManager.Insights is replaced by Azure.ResourceManager.Monitor and will be removed. The new track 2 package for machinelearningservices is in progress.

The breaking changes and other SDK failures are not cuased by the changes in this PR.

**Comment by raych1** [comment]
> The net track2 failure for insights and machinelearningservices are expected as these 2 are still the old track 2 packages. Azure.ResourceManager.Insights is replaced by Azure.ResourceManager.Monitor and will be removed. The new track 2 package for machinelearningservices is in progress.
> 
> The breaking changes and other SDK failures are not cuased by the changes in this PR.

@fengzhou-msft , the change looks good to me, do you think it's ready to merge? Are all of these RPs onboarded to track2 dotnet SDK?

File: specification/appconfiguration/resource-manager/readme.md [hunk 2]
```diff
@@ -115,7 +115,6 @@ swagger-to-sdk:
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-net-track2
   - repo: azure-sdk-for-java
-  - repo: azure-sdk-for-net
```
Comment (heaths) [inline specification/appconfiguration/resource-manager/readme.md:118]: There is a track 2 mgmt SDK, though: Azure.ResourceManager.AppConfiguration

File: specification/connectedvmware/resource-manager/readme.md (hunk 1 above)
Comment (heaths) [reply specification/connectedvmware/resource-manager/readme.md:60]: We do have a track 2 sdk\connectedvmwarevsphere\Azure.ResourceManager.ConnectedVMwarevSphere, though. Is this the same thing?

File: specification/dashboard/resource-manager/readme.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ This is not used by Autorest itself.
 
 ``` yaml $(swagger-to-sdk)
 swagger-to-sdk:
-  - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-net-track2
```
Comment (heaths) [inline specification/dashboard/resource-manager/readme.md:47]: I don't see any obvious service directory for this in Azure/azure-sdk-for-net. Is there even a track 2?

**Review by heaths — CHANGES_REQUESTED** [review]
A couple discrepancies based on what I see in teh Azure/azure-sdk-for-net repo in the `main` branch currently.

File: specification/dashboard/resource-manager/readme.md (hunk 3 above)
Comment (fengzhou-msft) [reply specification/dashboard/resource-manager/readme.md:47]: It's called grafana in azure-sdk-for-net as you can see the readme is used in https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/grafana/Azure.ResourceManager.Grafana/src/autorest.md

File: specification/appconfiguration/resource-manager/readme.md (hunk 2 above)
Comment (fengzhou-msft) [reply specification/appconfiguration/resource-manager/readme.md:118]: azure-sdk-for-net-track2 was configured before at line 116.

File: specification/connectedvmware/resource-manager/readme.md (hunk 1 above)
Comment (fengzhou-msft) [reply specification/connectedvmware/resource-manager/readme.md:60]: You're right. I somehow missed that due to the different service directory naming.

**Review by heaths — APPROVED** [review]

**Comment by fengzhou-msft** [comment]
> > The net track2 failure for insights and machinelearningservices are expected as these 2 are still the old track 2 packages. Azure.ResourceManager.Insights is replaced by Azure.ResourceManager.Monitor and will be removed. The new track 2 package for machinelearningservices is in progress.
> > The breaking changes and other SDK failures are not cuased by the changes in this PR.
> 
> @fengzhou-msft , the change looks good to me, do you think it's ready to merge? Are all of these RPs onboarded to track2 dotnet SDK?

@raych1 yes, these are for dotnet track 2 SDK automation. The 3 failed SDKs all have reasonable causes and will be fixed soon. This PR is ready to merge, thanks!

**Review by raych1 — APPROVED** [review]

**Comment by raych1** [comment]
@akning-ms , please force merge this PR as python/go sdk automation are timed out and not caused by this PR change.

**Pull request merged**